py/vtdb/field_types.py

```diff
@@ -93,7 +88,9 @@ def convert_bind_vars(bind_variables):
       new_vars[key] = times.DateTimeToString(val)
     elif isinstance(val, datetime.date):
       new_vars[key] = times.DateToString(val)
-    elif isinstance(val, (int, long, float, str, List, NoneType)):
+    elif isinstance(val, set):
+      new_vars[key] = list(val)
```

Comment:
To be deterministic, I would prefer sorted(val) over list(val), here and elsewhere. All set elements are comparable, right?